File: src/0961-N-Repeated-Element-in-Size-2N-Array/0961.cpp
static int x = []() {std::ios::sync_with_stdio(false); cin.tie(0); return 0; }();
class Solution 
{
public:
    int repeatedNTimes(vector<int>& A) 
    {
        for (int i = 0; i < A.size()-1; ++i)
        {
            if ((A[0] == *(A.rbegin())) or (A[0] == *(A.rbegin() + 1))) return A[0];
            else if (A[1] == *(A.rbegin())) return A[1];
            else if (A[i] == A[i+1] or (i < A.size() -2 and A[i] == A[i + 2])) return A[i];
        }
    }
};

File: src/0961-N-Repeated-Element-in-Size-2N-Array/0961.py
class Solution:
    def repeatedNTimes(self, A):
        """
        :type A: List[int]
        :rtype: int
        """
        for i in range(len(A)):
            if A[i - 1] == A[i] or A[i - 2] == A[i]:
                return A[i]